From 29959d57f120071a4522d95633446052b78f0b90 Mon Sep 17 00:00:00 2001
From: Carsten Haitzler (Raster <raster@rasterman.com>
Date: Thu, 30 Oct 2008 14:11:08 +1100
Subject: [PATCH] edje: disable program cache as it uses a fair bit of ram

disable the program cache as it uses a fair bit of ram and you probably
should re-evaluate your usage of edje if you need it for speed.
---
 packages/efl1/edje_svn.bb |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/packages/efl1/edje_svn.bb b/packages/efl1/edje_svn.bb
index 17c8f91..5cbeec9 100644
--- a/packages/efl1/edje_svn.bb
+++ b/packages/efl1/edje_svn.bb
@@ -6,8 +6,9 @@ PR = "r5"
 
 inherit efl
 
-# gain some extra performance at the expense of RAM
-EXTRA_OECONF = "--enable-edje-program-cache"
+# gain some extra performance at the expense of RAM - generally i'd say bad
+# and a possible source of bugs
+#EXTRA_OECONF = "--enable-edje-program-cache"
 
 PACKAGES =+ "${PN}-utils"
 RDEPENDS_${PN}-utils = "cpp cpp-symlinks embryo-tests"
-- 
1.5.4.3

